http/tests/raids.rs

```diff
@@ -0,0 +1,50 @@
+#![cfg(feature = "blocking")]
+
+use gw2lib::{
+    model::misc::raids::{Raid, RaidId, Wing},
+    Requester,
+};
+
+pub mod setup;
+
+#[test]
+fn all() {
+    let client = crate::setup::setup();
+    let _: Vec<Raid> = client.all().unwrap();
+}
+
+#[test]
+fn ids() {
+    let client = crate::setup::setup();
+    let _: Vec<RaidId> = client.ids::<Raid, _>().unwrap();
+}
+
+#[test]
+#[ignore]
+fn forsaken_thicket_wings() {
```

Comment:
tests should only be ignored if they take minutes to complete. These tests here should always run.